tests/Unit/NumericalAlgorithms/Integration/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# Distributed under the MIT License.
```

Comment:
This file isn't read by cmake, which means the test is never compiled. Please add the line `add_subdirectory(Integration)` to the `CMakeLists.txt` in the directory one up.

Reply:
Done.